pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-inject-parameters.adoc

```diff
@@ -67,4 +72,221 @@ You can use the following function definition:
 ----
 let filter = on_request(request_filter)
         .on_response(response_filter);
-----
\ No newline at end of file
+----
+
+[[metadata]]
+== Metadata
+
+PDK exposes a `Metadata` injectable that provides metadata about the Policy, the Flex instance, the API instance and the Anypoint Organization. The `Metadata` struct is as follows:
+
+[source,Rust]
+----
+pub struct Metadata {
+    pub flex_metadata: FlexMetadata,
+    pub policy_metadata: PolicyMetadata,
+    pub api_metadata: ApiMetadata,
+    pub platform_metadata: PlatformMetadata,
+}
+----
+
+The `Metadata` struct references the following structures:
+
+* `FlexMetadata`:
++
+[source,Rust]
+----
+pub struct FlexMetadata {
+    pub flex_name: String,
+    pub flex_version: String,
+}
+----
+
+* `PolicyMetadata`:
++
+[source,Rust]
+----
+pub struct PolicyMetadata {
+    pub policy_name: String,
+    pub policy_namespace: String,
+}
+----
+
+* `ApiMetadata`:
++
+[source,Rust]
+----
+pub struct ApiMetadata {
+    pub id: Option<String>,
+    pub name: Option<String>,
+    pub version: Option<String>,
+    pub slas: Option<Vec<ApiSla>>, // API SLA tiers in the platform
+}
+
+// Each ApiSla contains the limits for each SLA. 
+pub struct ApiSla {
+    pub id: String,
+    pub tiers: Vec<Tier>, // Limits within an API SLA tier in the platform
+}
+----
+
+* `PlatformMetadata`:
++
+[source,Rust]
+----
+pub struct PlatformMetadata {
+    pub organization_id: String,
+    pub environment_id: String,
+    pub root_organization_id: String,
+}
+----
+
+To inject metadata into the wrapped functions:
```

Comment:
Maybe we could change how we expose this example. The example below shows how to use the metadata inside a Filter function. But it is also completely valid to use the metadata just in the entrypoint function.

So I wouldn't mention this examples as the only recipe on how to use metadata, but just as what it is, an example of usage.